app/routes/signatures/ongoing.js

```diff
@@ -50,8 +62,17 @@ export default class SignaturesOngoingRoute extends Route {
     if (this.currentSession.may('view-all-ongoing-signatures')) {
       filter[':has:creator'] = 't';
     } else {
+      // User has a common organisation with the creator of the flow
       filter['creator'] = {
-        ':id:': this.currentSession.user.id,
+        memberships: {
+          organization: {
+            memberships: {
+              user: {
+                ':id:': this.currentSession.user.id,
+              },
+            },
+          },
+        },
```

Comment:
I'd replace this with

```js
memberships: {
  organization: {
    ':id:': this.currentSession.organization.id,
  }
}
```

Locally it seems to work, I don't immediately see a reason why the extra hop back to the current user is needed.